server/service/integration_mdm_test.go

```diff
@@ -7249,6 +7249,650 @@ func (s *integrationMDMTestSuite) TestHostDiskEncryptionKey() {
 	require.Equal(t, "DEF", string(decrypted))
 }
 
+// ///////////////////////////////////////////////////////////////////////////
+// Common MDM config test
+
+func (s *integrationMDMTestSuite) TestMDMEnabledAndConfigured() {
+	t := s.T()
+	ctx := context.Background()
+
+	appConfig, err := s.ds.AppConfig(ctx)
+	originalCopy := appConfig.Copy()
+	require.NoError(t, err)
```

Comment:
Nit: this should go above the `originalCopy` line, so that it doesn't panic but fails with the proper error if line 7259 results in an error.